docs/source/DeveloperGuide/AddingNewComponents/Utilities/gLogger/gLogger/Basics/index.rst

```diff
@@ -0,0 +1,607 @@
+
+Here is a component section which contains *Logging* and *Backend*
+configuration:
+
+::
+
+    Agents
+    {
+        SimplestAgent
+        {
+          LogLevel = INFO
+          LogBackends = stdout,stderr,file
+          BackendsOptions
+          {
+            FileName = /tmp/logtmp.log
+          }
+          LogColor = False
+          LogShowLine = True
+        }
+    }   
+
+To summarize, this file configures an agent named *SimplestAgent*, sets
+the level of *gLogger* at *info*, adds 3 *Backend* objects to it, which
+are *stdout*, *stderr* and *file*. Thus, each log record superior to
+*info* level, created by a *Logging* object in the agent, will be send
+to 3 different outputs. We learn also from the *BackendOptions* that the
+*file Backend* will send these log records to the */tmp/logtmp.log*
+file.
+
+In addition, the log records will be not displayed with color, and the
+caller path name will not appear if we do not change the level to
+*debug*.
+
+Summary of the command line argument configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Here is a complete table explaining the changes provided by the command
+line argument *-d*: 
+
++--------------------------------------+----------------+----------------+-----------+
+| Argument                             | ShowHeader     | showThread     | Level     |
++======================================+================+================+===========+
+| Default(Executors/Agents/Services)   | True           | False          | Notice    |
++--------------------------------------+----------------+----------------+-----------+
+| Default(Scripts)                     | False          | False          | Notice    |
++--------------------------------------+----------------+----------------+-----------+
+| -d                                   | DefaultValue   | DefaultValue   | Verbose   |
++--------------------------------------+----------------+----------------+-----------+
+| -dd                                  | True           | DefaultValue   | Verbose   |
++--------------------------------------+----------------+----------------+-----------+
+| -ddd                                 | True           | True           | Debug     |
++--------------------------------------+----------------+----------------+-----------+
+
+About multiple processes and threads
+------------------------------------
+
+Multiple processes
+~~~~~~~~~~~~~~~~~~
+
+*DIRAC* is composed by many micro services running in multiple
+processes. *gLogger* object is naturally different for two distinct
+processes and can not save the application from process conflicts.
+Indeed, *gLogger* is not process-safe, that means that two processes can
+encounter conflicts if they try to write on a same file at the same
+time. So, be careful to avoid the case.
+
+Multiple threads
+~~~~~~~~~~~~~~~~
+
+*gLogger* is based on the Python *logging* library which is completely
+thread-safe. Thus, it is also thread-safe.
```

Comment:
Watch out ! That is a very daring statement ! In fact, I am not sure your code is thread safe at all !